src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -20,17 +20,16 @@
 
 #include "src/core/ext/filters/client_channel/client_channel.h"
 
+#include <grpc/support/alloc.h>
+#include <grpc/support/log.h>
+#include <grpc/support/string_util.h>
+#include <grpc/support/sync.h>
 #include <inttypes.h>
 #include <limits.h>
 #include <stdbool.h>
 #include <stdio.h>
 #include <string.h>
 
-#include <grpc/support/alloc.h>
-#include <grpc/support/log.h>
-#include <grpc/support/string_util.h>
-#include <grpc/support/sync.h>
-
```

Comment:
Can you put the headers back the way they were here? These aren't really system headers so it looks odd.

Reply:
thank you for catching this. I think the script has done this. reverted.